images/bootstrap/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 # Includes basic workspace setup, with gcloud and a bootstrap runner
 
-FROM debian:bullseye
+FROM debian:bookworm
```

Comment:
we'll need to keep an eye on how CI goes when this filters into kubekins-e2e etc because of e.g. docker getting upgraded